File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/ClienteDAO.java
package pe.edu.pucp.softinv.dao;

import pe.edu.pucp.softinv.model.Personas.ClienteDTO;
import pe.edu.pucp.softinv.model.Personas.UsuarioDTO;

public interface ClienteDAO {
    Integer insertar(ClienteDTO usuario);
    ClienteDTO obtenerPorId(Integer id);
    Integer modificar(ClienteDTO usuario);
    Integer eliminar(ClienteDTO id);
}

File: C#/SoftInvPersistance/DAO/ClienteDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    public interface ClienteDAO
    {
        int insertar(ClienteDTO usuario);
        ClienteDTO obtenerPorId(int id);
        int modificar(ClienteDTO usuario);
        int eliminar(ClienteDTO cliente);
    }
}
